opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.hpp

```diff
@@ -46,8 +46,11 @@ namespace Opm {
             const_iterator begin() const;
             const_iterator end() const;
 
+            static std::vector <std::string> getAllExpandedKeywords();
```

Comment:
Then it shouldn't be a member at all, but rather a static var in `SummaryConfig.cpp`.